src/components/MetaData.svelte

```diff
@@ -0,0 +1,11 @@
+<script>
+  export let data;
+</script>
+
+{#if data}
+  <div
+    class="inline-block border border-blue-500 rounded-full text-xs py-1 px-3 w-15 bg-blue-800 text-white">
```

Comment:
We should make the background/border colours a configurable property of the component, rather than hardcoding it here.